### ISSUE TITLE
feature: output directory

### ISSUE THREAD

**Issue by fr-olivier** [simonhaenisch/md-to-pdf#133]
Hello,

Is it possible to specify a folder path where all the generated pdfs would be exported?

**Comment by fr-olivier** [comment]
Ok thanks for your answer. If this is added as a new feature in the future, is there a way to be informed about it?

**Comment by simonhaenisch** [comment]
Yeah I'll leave this issue open as a feature request so as long as you're subscribed you'll know when it gets closed.
